utils/utils.py

```diff
@@ -82,7 +82,8 @@ def save_options(options, save_dir):
     for option in vars(options):
         opt_dict[option] = getattr(options, option)
 
-    mkdir(save_dir)
+    if not os.path.exists(save_dir):
+        mkdir(save_dir)
```

Comment:
`mkdir` has a check built into it, so you don't need to check before calling the function

Reply:
ah true, will edit it